kubectl-fdb/cmd/cordon_test.go

```diff
@@ -100,41 +101,80 @@ var _ = Describe("[plugin] cordon command", func() {
+					customLabel:    fdbv1beta2.FDBClusterLabel,
 				}),
 			Entry("Cordon node without exclusion",
 				testCase{
 					nodes:                     []string{"node-1"},
 					WithExclusion:             false,
 					ExpectedInstancesToRemove: []string{},
 					ExpectedInstancesToRemoveWithoutExclusion: []string{"instance-1"},
+					useCustomLabel: false,
+					customLabel:    fdbv1beta2.FDBClusterLabel,
 				}),
 			Entry("Cordon no nodes with exclusion",
 				testCase{
 					nodes:                     []string{""},
 					WithExclusion:             true,
 					ExpectedInstancesToRemove: []string{},
 					ExpectedInstancesToRemoveWithoutExclusion: []string{},
+					useCustomLabel: false,
+					customLabel:    fdbv1beta2.FDBClusterLabel,
 				}),
 			Entry("Cordon no node nodes without exclusion",
 				testCase{
 					nodes:                     []string{""},
 					WithExclusion:             false,
 					ExpectedInstancesToRemove: []string{},
 					ExpectedInstancesToRemoveWithoutExclusion: []string{},
+					useCustomLabel: false,
+					customLabel:    fdbv1beta2.FDBClusterLabel,
 				}),
 			Entry("Cordon all nodes with exclusion",
 				testCase{
 					nodes:                     []string{"node-1", "node-2"},
 					WithExclusion:             true,
 					ExpectedInstancesToRemove: []string{"instance-1", "instance-2"},
 					ExpectedInstancesToRemoveWithoutExclusion: []string{},
+					useCustomLabel: false,
+					customLabel:    fdbv1beta2.FDBClusterLabel,
 				}),
 			Entry("Cordon all nodes without exclusion",
 				testCase{
 					nodes:                     []string{"node-1", "node-2"},
 					WithExclusion:             false,
 					ExpectedInstancesToRemove: []string{},
 					ExpectedInstancesToRemoveWithoutExclusion: []string{"instance-1", "instance-2"},
+					useCustomLabel: false,
+					customLabel:    fdbv1beta2.FDBClusterLabel,
+				}),
+			Entry("Cordon node with custom label without exclusion",
+				testCase{
+					nodes:                     []string{"node-1"},
+					WithExclusion:             true,
+					ExpectedInstancesToRemove: []string{"instance-1"},
+					ExpectedInstancesToRemoveWithoutExclusion: []string{},
+					useCustomLabel: true,
+					customLabel:    fdbv1beta2.FDBClusterLabel,
+				}),
+			Entry("Cordon nodes with custom label without exclusion",
+				testCase{
+					nodes:                     []string{"node-1", "node-2"},
+					WithExclusion:             false,
+					ExpectedInstancesToRemove: []string{},
+					ExpectedInstancesToRemoveWithoutExclusion: []string{"instance-1", "instance-2"},
+					useCustomLabel: true,
+					customLabel:    fdbv1beta2.FDBClusterLabel,
+				}),
+			Entry("Cordon no nodes with exclusion with custom label",
+				testCase{
+					nodes:                     []string{""},
+					WithExclusion:             true,
+					ExpectedInstancesToRemove: []string{},
+					ExpectedInstancesToRemoveWithoutExclusion: []string{},
+					useCustomLabel: true,
+					customLabel:    fdbv1beta2.FDBClusterLabel,
```

Comment:
Can we add some test cases where we remove multiple Pods from different clusters? The current test case is not enough to test the behaviour with multiple Pods from different clusters running on the same node.